README.md

```diff
@@ -1,93 +1,200 @@
+
+### 2. Generating Elastic IP
+Next, we need to set an Elastic IP for our Host VM. Elastic IPs are just static IPs designed for dynamic cloud computing.
+
+1. Install/Upgrade AWS CLI to the latest version. If you need help installing the AWS CLI,
+please click [here](https://docs.aws.amazon.com/cli/latest/userguide/installing.html).
+1. If you don't know your AWS Access keys and Secret Keys, please go [here](https://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html) to setup them up.
+Otherwise, skip to the next step.
+1. Run `aws configure` and input your...
+    - AWS access key
+    - AWS secret key
+    - default region name (The Region of Your Host VM)
+    - default output format (Output format is usually `json`)
+    
+1. Run
+   ```
+   aws ec2 allocate-address --domain "vpc" --region <region name>
+   ```
+1. You should receive a text response like the one below.
+    ```json
+    {
+        "PublicIp": "99.99.999.999",
+        "AllocationId": "eipalloc-12345678",
+        "Domain": "vpc"
+    }
+    ```
+    Record the `PublicIP` in this response. This will be used for our Host VM.
+1. Go back to the AWS Console.
+1. Select the top toolbar. Click **Service**. Then, click **EC2**.
+1. In the sidebar under the **Network & Security** subsection, click the **Elastic IPs** link.
+1. In the table, find and select the Elastic IP you generated earlier.
+1. Click in the **Actions** button on top of the table. Then, click **Associate Address**.
+1. In the Instance textbox, put in the name of your Host VM.
+1. Afterwards, click the caret on the **Private IP** textbox. There should be only one ip in the dropdown.
+   That is your Host VMs private IP. Click on that IP.
+1. Finally, click **Associate**.
+
+### 3. Add remaining settings to Security Group
+Now that we created our Elastic IP. We can finish configuring our Security Group. 
+
+1. Add settings in the table below to your own securty group.
+
+|Type|Protocol|Port Range|Source|
+|----|--------|----------|------|
+|ALL TCP|TCP|0 - 65535|\<Elastic IP>|
+|ALL TCP|TCP|0 - 65535|\<Name of your Security Group>|
+
+### 4. Create your Host VM's Subdomain
+Now we need to create a subdomain and link it to for Host VM. Your subdomain should contain your name.
+So if your name is Bob, your subdomain should be `bob.ucsc-cgp-dev.org`.
+
+1. Select the top toolbar. Click **Services**. Then, click **Route 53**.
+1. Click on **Hosted Zones** in the sidebar.
+1. In the table, click on **ucsc-cgp-dev.org.** under the **Domain Name** column.
+1. Click on the Blue **Create Record Set** button.
+1. A form should appear on the right side of the screen. Put in your name in the **Name** text box in the form.
+1. In the **Value** text box, put in the elastic IP of your host VM.
+ 
+## Deploying the Platform
+
+### 1. Before Installation
+
+1. Before you start deploying make sure the following components are already setup
+    - DOS Azul Lambda
```

Comment:
Please reorder as follows (the (partial) ordering in which they would need to be installed):
- HCA/DataBiosphere Data Storage System(DSS)
- DataBiosphere Azul indexer (for Commons)
- DOS Azul Lambda
- Boardwalk [README](boardwalk/README.md) for instructions on how to setup the external components.
- Bagit Firecloud Lambda